### ISSUE TITLE
abp4.4 can't many to many

### ISSUE THREAD

**Issue by ma7x** [abpframework/abp#9834]
in the 4.3.3
`public DbSet<AppUser> Users { get; set; }`
and in the 4.4
`public DbSet<IdentityUser> Users { get; set; }`

I want many to many table User and GameRound, AbpUser entity is gone

```
public class GameRound : FullAuditedAggregateRoot<Guid>
    {
        public string Name { get; set; }
        public RoundStatus Status { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime? EndTime { get; set; }
        public string Remark { get; set; }

        public List<GameRoundUser> GameRoundUsers { get; set; }
    }


public class GameRoundUser
    {
        public Guid UserId { get; set; }
        public IdentityUser User { get; set; }

        public Guid GameRoundId { get; set; }
        public GameRound GameRound { get; set; }
    }

```
protected override void OnModelCreating(ModelBuilder builder)
```

builder.Entity<GameRoundUser>()
                .HasOne(pt => pt.User)
                .WithMany("GameRoundUsers")
                .HasForeignKey(pt => pt.UserId);
            builder.Entity<GameRoundUser>()
                .HasOne(pt => pt.GameRound)
                .WithMany(t => t.GameRoundUsers)
                .HasForeignKey(pt => pt.GameRoundId);
```
then add-migration 
`
The entity type 'GameRoundUser' requires a primary key to be defined. If you intended to use a keyless entity type, call 'HasNoKey' in 'OnModelCreating'. For more information on keyless entity types, see https://go.microsoft.com/fwlink/?linkid=2141943.`
I search documentation 
https://docs.abp.io/en/abp/latest/Customizing-Application-Modules-Extending-Entities
but id dit not work
#my question  is how to many-to-many relationship table IdentityUser and GameRound

**Comment by realLiangshiwei** [comment]
I don't understand what you mean.

**Comment by ma7x** [comment]
> I don't understand what you mean.

how many-to-many relationship IdentityUser in abp4.4

**Comment by ma7x** [comment]
I fix it  thanks  #1517
